simple_history/tests/models.py

```diff
@@ -983,3 +987,58 @@ class TestHistoricParticipanToHistoricOrganization(models.Model):
         related_name="historic_participants",
     )
     history = HistoricalRecords()
+
+
+class TestParticipantToHistoricOrganizationOneToOne(models.Model):
+    """
+    Non-historic table foreign key to historic table.
+
+    In this case it should simply behave like ForeignKey because
+    the origin model (this one) cannot be historic, so foreign key
+    lookups are always "current".
```

Comment:
```suggestion
    In this case it should simply behave like OneToOneField because
    the origin model (this one) cannot be historic, so one to one field
    lookups are always "current".
```